src/app-layout/skeleton/layout.tsx

```diff
@@ -0,0 +1,78 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import { AppLayoutPropsWithDefaults } from '../interfaces';
+import styles from './styles.css.js';
+import customCssProps from '../../internal/generated/custom-css-properties';
+import clsx from 'clsx';
+
+const contentTypeCustomWidths: Array<string | undefined> = ['dashboard', 'cards', 'table'];
+
+interface SkeletonLayoutProps
+  extends Pick<
+    AppLayoutPropsWithDefaults,
+    | 'notifications'
+    | 'contentHeader'
+    | 'content'
+    | 'contentType'
+    | 'minContentWidth'
+    | 'maxContentWidth'
+    | 'disableContentPaddings'
+    | 'navigation'
+    | 'navigationWidth'
+    | 'tools'
+    | 'toolsWidth'
+    | 'placement'
+  > {
+  topBar?: React.ReactNode;
+}
+
+export function SkeletonLayout({
+  notifications,
+  contentHeader,
```

Comment:
Just to make sure, we want to support `contentHeader` here although it is deprecated in App Layout?

Reply:
We release this feature as a non-breaking change, so we are required to support existing slots